### PR TITLE
Typescript info.js

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

**Comment by Zarel** [comment]
I don't like the custom pokedex in Gen 3. I'd much rather do it in `getSpecies`.

**Comment by Zarel** [comment]
The validator is just hardcoded to consider Gen 4 abilities invalid in Gen 3. I've given up on a lot of that premature optimization these days but I still think updating `getSpecies` is a lot better than individually updating the dex entries.

File: data/mods/gen3/scripts.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ let BattleScripts = {
 	gen: 3,
 	init() {
 		for (let i in this.data.Pokedex) {
-			delete this.data.Pokedex[i].abilities['H'];
```
Comment (Zarel) [inline data/mods/gen3/scripts.js:9]: I assume this means you tested and found this unnecessary?

Comment (KrisXV) [reply data/mods/gen3/scripts.js:9]: yes

**Comment by KrisXV** [comment]
I accidentally pushed to this pull request, but the code was completely reset to master before I started typescripting so this can just be repurposed

**Pull request closed without merging**